pkg/controller/config.go

```diff
@@ -11,12 +11,20 @@ type Config struct {
 	Debug    bool `envconfig:"DEBUG"`
 	HTTPPort int  `envconfig:"WEB_HTTP_PORT" default:"8080"`
 	Logger   observability.LoggerConfig
+
 	// CleanUpThresholdEnvVar is used if we want to increase the amount of time a
 	// load test lives for, the default is 1 hour. (ex. 5h)
 	CleanUpThreshold time.Duration `envconfig:"CLEANUP_THRESHOLD" default:"1h"`
+
 	// S3 compatible configuration access keys and endpoints needed to store load test reports
 	KangalProxyURL string `envconfig:"KANGAL_PROXY_URL" default:""`
 
+	// KubeClientTimeout specifies timeout for each operation done by kube client
+	KubeClientTimeout time.Duration `envconfig:"KUBE_CLIENT_TIMEOUT" default:"5s"`
+
+	// SyncHandlerTimeout specifies the time limit for each sync operation
+	SyncHandlerTimeout time.Duration `envconfig:"SYNC_HANDLER_TIMEOUT" default:"60s"`
```

Comment:
What do you thing about logging a warning/error when `SyncHandlerTimeout` is less then `KubeClientTimeout`?

Reply:
I think it would be helpful, but should be done in another pull-request (perhaps to also include checks/warnings for other configs) so it can be planned/reviewed better.

Reply:
:+1: Can you please create a ticket for this